rest/blip_sync.go

```diff
@@ -102,17 +102,19 @@ func (sc *ServerContext) incrementConcurrentReplications(ctx context.Context) (b
 		return false, base.ErrReplicationLimitExceeded
 	}
 	sc.ActiveReplicationsCounter.activeReplicatorCount++
+	db.DbStats.Database().NumConcurrentReplications.Add(1)
```

Comment:
move outside the lock, since this is an atomic that doesn't require the `ActiveReplicationsCounter` lock.